### PR TITLE
Add "thanks" and "thank you" to "pray" emoji

### PR DESCRIPTION
**Pull request by esethna**
Added short names for the pray emoji (thanks, and thank you)

```release-note
Added thanks and thank you for pray emoji
```

### CODE REVIEW

**Comment by M-ZubairAhmed** [comment]
Git Diff failed, i am going to make another PR to make it easier to see the difference

**Comment by M-ZubairAhmed** [comment]
We also have to run `npm run make-emojis` for the new data to get included in the build

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
We also have to run `npm run make-emojis` for the new data to get included in the build

**Review by M-ZubairAhmed — APPROVED** [review]
@esethna please verify once

**Comment by esethna** [comment]
@M-ZubairAhmed thanks, looks great!

File: webapp/channels/build/emoji/additional_shortnames.json [hunk 1]
```diff
@@ -2200,6 +2200,10 @@
     "om_symbol":[
         "om"
     ],
+    "pray": [
+        "thanks",
+        "thank you"
```
Comment (yasserfaraazkhan) [inline webapp/channels/build/emoji/additional_shortnames.json:2205]: @M-ZubairAhmed should it be `thank_you`
in the spinwick, we are only seeing `:thanks`

Comment (esethna) [reply webapp/channels/build/emoji/additional_shortnames.json:2205]: @yasserfaraazkhan it should be "thank you" with a space. It was working for me on the spinwick

Comment (M-ZubairAhmed) [reply webapp/channels/build/emoji/additional_shortnames.json:2205]: @esethna can you check again even because emoji names don't have spaces

**Comment by larkox** [comment]
Does this require mobile changes? Or are they on the way?

File: webapp/channels/build/emoji/additional_shortnames.json (hunk 1 above)
Comment (esethna) [reply webapp/channels/build/emoji/additional_shortnames.json:2205]: image

@M-ZubairAhmed

**Comment by esethna** [comment]
@larkox good question, are mobile changes needed? @M-ZubairAhmed perhaps you would know

**Comment by M-ZubairAhmed** [comment]
> @larkox good question, are mobile changes needed? @M-ZubairAhmed perhaps you would know

Yes it would need mobile change, are you open to creating a ticket for it

File: webapp/channels/build/emoji/additional_shortnames.json (hunk 1 above)
Comment (M-ZubairAhmed) [reply webapp/channels/build/emoji/additional_shortnames.json]: @yasserfaraazkhan i added both `thank you` and `thank_you`

**Comment by esethna** [comment]
> Yes it would need mobile change, @esethna are you open to creating a ticket for it

Created here: https://mattermost.atlassian.net/browse/MM-54934

Thanks all.

**Review by yasserfaraazkhan — APPROVED** [review]

**Comment by yasserfaraazkhan** [comment]
image

**Pull request merged**